.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/ocornut/imgui
 [submodule "examples/third-party/ggsock"]
 	path = examples/third-party/ggsock
-	url = https://github.com/ggerganov/ggsock
+	url = git@github.com:guilt/ggsock.git
```

Comment:
We can first merge the `ggsock` changes and then keep the submodule pointing to https://github.com/ggerganov/ggsock